client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -176,6 +183,26 @@ const ScreenMain = ( {
 							</NavigationButtonAsItem>
 						</>
 					</NavigatorItemGroup>
+					{ ! isNoticeDismissed && (
+						<Notice className="screen-container__notice" onRemove={ dismissNotice }>
+							{ translate(
+								'After activation, your homepage content will be replaced. But you can still access your old content. {{a}}Learn more{{/a}}.',
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Am I correct in understanding what actually will happen is that "your _pages/posts_ will persist, but your _contents in any templates_ will be deleted"?

If so,

>After activation, your homepage content will be replaced. But you can still access your old content. {{a}}Learn more{{/a}}.

Wouldn't this copy mislead people into thinking that all content is preserved? (I'm sorry If I'm wrong 🙏 )

Reply:
I now understand what will happen. Thank you!